experimental/examples/minesweeper/build.gradle.kts

```diff
@@ -54,29 +47,36 @@ kotlin {
             }
         }
     }
+
+    // Workaround for an issue:
+    //    https://youtrack.jetbrains.com/issue/KT-53561/Invalid-LLVM-module-inlinable-function-call-in-a-function-with-debug-info-must-have-a-dbg-location
+    // Compose compiler produces nodes without line information sometimes that provokes Kotlin native compiler to report errors.
+    // TODO: remove workaround when switch to Kotlin 1.8
+    val disableKonanVerification = "-Xverify-compiler=false"
```

Comment:
Why is it "konan"? :)

Reply:
Konan is short name for Kotlin/Native compiler